packages/teleport-types/src/generators.ts

```diff
@@ -154,6 +162,13 @@ export interface ProjectStrategy {
   }
 }
 
+export interface CustomTag {
```

Comment:
can we reuse the code from CustomLinkTag? or use this for CustomLinkTag so we don't duplicate?

Reply:
Yes, had it in my list. We can merge all three and make one